api/src/main/java/org/openmrs/module/pihcore/setup/ConfigurationSetup.java

```diff
@@ -174,6 +171,14 @@ public void configureNonConceptDependencies() throws Exception {
         setStatus("Loading initializer pre-concept domains");
         InitializerSetup.loadPreConceptDomains(config);
 
+        // install concepts from OCL
+        setStatus("Installing Concepts from OCL zip");
+        InitializerSetup.installDomain(OCL, config);
+
+        // Load remaining Initializer domains that could depend on Concepts
+        setStatus("Loading initializer post-concept domains");
+        InitializerSetup.loadPostConceptDomains(config);
```

Comment:
At this point, we don't need these 3 steps at all @mogoodrich .  We should just call a single `InitializerSetup.install(config)` method that just installs from Iniz in the default order that Iniz specifies (we don't need any custom domain arrays to load in a specific order), and just applies our custom logic about site-specific configs and re-loading some domains if others have changed (eg. location / location tag).

Basically, convert `loadPreConceptDomains` into an `install` method, and remove the steps that specifically exclude `ocl` or anything in the `getDomainsToLoadAfterConcepts` list, which can be totally removed, but which keeps the logic that excludes files meant for a specific site.

Reply:
Right, duh, thanks @mseaton  for the step-by-step details of what to change.  I just reworked and confirmed that a HUMCI SDK installed without error.  Will test the others after your review.